config/core/resources/pingsource.yaml

```diff
@@ -57,88 +57,6 @@ spec:
   - name: v1alpha1
     served: true
     storage: true
-    schema:
```

Comment:
Where do you actually say to do the conversion?

Reply:
It happens next https://github.com/knative/eventing/pull/2619